modules/ebs__enum_volumes_snapshots/main.py

```diff
@@ -90,7 +90,7 @@ def main(args, pacu_main):
         ec2_data['Volumes'] = []
     if 'Snapshots' not in ec2_data.keys():
         ec2_data['Snapshots'] = []
-    session.update(pacu_main.database, EC2=ec2_data)
+    session.json_update(EC2=ec2_data)
```

Comment:
json_update behaves the same way as the previous session.update, which handled extra json marshalling.